maturity-level-one/src/README.md

```diff
@@ -1,67 +1,98 @@
 # Practical API Guidelines - Maturity level One "Must have" - Code Samples
```

Comment:
Feel free to remove all spaces before the content of every section as it's now rendered as code

Reply:
How would you set the summary for the 3 updates (1 PUT, 2 PATCH) ? 
It's a technical difference.... mmmm... what to put as business oriented description? 
        /// <summary>
        /// Update player profile (one/two/etc...)
        /// </summary>

Reply:
They can be the same if you want, the operationId should still be different.

In terms of description they can be different in the sense that they can mention incremental updates or not

Reply:
done